index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Porfolio Project</title>
+</head>
+<body>
+    <style>
+        body{
+            background-image: url(img/header-shapes\ mobile.png);
+        }
+    </style>
+
+    <header>
+        <nav>
+            <ul  class="navbar">
+                <li><a class="logo" href="">My Logo</a></li>
+                <li> <a href=""><img class="toggler-icon" src="img/hamburger-icon.png" alt="Hamburger Icon"></a></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="container">
+        <h1 id="intro">I'm Maraine<br>Glad to see you!</h1>
+        <p id="intro-details">I’m a software developer!
+            I can <br> help you build a product, 
+            feature or website <br> Look 
+            through some of my work and <br> experience! 
+            If you like what you see and <br> have a project you need coded, 
+            don’t <br> hestiate to contact me.</p>
+    </div>
+    <section class="connect">
+        <h3 id="intro-connect">LET'S CONNECT</h3>
+        <ul class="socials">
+            <li><a href=""><img src="img/twitter-icon.png" alt=""></a></li>
+            <li><a href=""><img src="img/linkedin-icon.png" alt="linkedin"></a></li>
+            <li><a href=""><img src="img/medium-icon.png" alt=""></a></li>
+            <li><a href=""><img src="img/github-icon.png" alt=""></a></li>
+            <li><a href=""><img src="img/angellist-icon.png" alt=""></a></li>
+        </ul>
+    </section>
```

Comment:
- [OPTIONAL] I suggest you wrap this Headline section in a `<section>` tag and change the 'connect' section you wrapped in a `<section>` tag to a `<div>` tag for better semantics. Please read [this article for better understanding](https://www.w3schools.com/html/html5_semantic_elements.asp).